app/views/vm_common/_disassociate_floating_ip.html.haml

```diff
@@ -7,14 +7,15 @@
       %label.col-md-2.control-label
         = _('Floating IP')
       .col-md-8
-        %select{:name      => 'floating_ip',
-                'ng-model' => 'vmCloudModel.floating_ip',
-                'ng-options' => 'floating_ip.address for floating_ip in floating_ips track by floating_ip.address'}
+        %select{:name        => 'floating_ip',
+                'ng-model'   => 'vm.vmCloudModel.floating_ip',
+                'ng-options' => 'floating_ip.address for floating_ip in vm.floating_ips track by floating_ip.address'}
 
   %div_for_paging{'ng-controller'                    => "pagingDivButtonGroupController",
                   'paging_div_buttons_id'            => "angular_paging_div_buttons",
                   'paging_div_buttons_type'          => "Submit"}
 
+
 - unless @explorer
```

Comment:
You want to move this to the right too, otherwise it's not part of the form and can't access the controller..